### PR TITLE
Docs: Canned trigger element replacement info

### PR DESCRIPTION
**Pull request by JLGarber**
First pass at documenting this. I think this is a reasonable balance of information about these elements while leaving most of the internals in the respective files. Please comment and improve!

### CODE REVIEW

File: docs/RaidbossGuide.md [hunk 1]
```diff
@@ -187,6 +187,70 @@ Trigger elements are evaluated in this order, and must be listed in this order:
...
+  regexFr: Regexes.startsUsing({ source: 'Primo-Alexander', id: '4A83', capture: false }),
+  regexJa: Regexes.startsUsing({ source: 'アレキサンダー・プライム', id: '4A83', capture: false }),
+  regexCn: Regexes.startsUsing({ source: '至尊亚历山大', id: '4A83', capture: false }),
+  regexKo: Regexes.startsUsing({ source: '알렉산더 프라임', id: '4A83', capture: false }),
+  condition: Conditions.caresAboutMagical(),
+  response: Responses.bigAoe('alert'),
+},
+```
+
+While this doesn't reduce the number of lines we need to match the locale regexes, this is far less verbose than:
```
[the remark this answers is not in the document]
Comment (JLGarber) [reply docs/RaidbossGuide.md]: Definitely worth an issue. I don't think it's something I can do myself, but I can discuss the bike shed. I would very much like to get away from six lines where we really should need only one.

**Comment by JLGarber** [comment]
No worries on style things, documentation really needs to be as clean and style-unified as possible. I'm puzzled how I totally missed full stops when I went through to check for line breaks, I think I correctly broke at all the commas.

**Comment by JLGarber** [comment]
Oh, I guess I should link it here as well: Should resolve #1286.

File: docs/RaidbossGuide.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (quisquous) [reply docs/RaidbossGuide.md:227]: Filed #1306

**Pull request merged**